File: Programmers/Lv.0/가까운 수/120890.java
import java.util.Arrays;

class Solution {
    public int solution(int[] array, int n) {
        Arrays.sort(array);
        
        int answer = array[0];
        int mindiff = Math.abs(answer - n);
        
        for (int i=1; i<array.length; i++) {
            int diff = Math.abs(array[i] - n);
            if (diff < mindiff) {
                answer = array[i];
                mindiff = diff;
            }
        }
        
        return answer;
    }
}

File: Programmers/Lv.0/가까운 수/120890.js
function solution(array, n) {
    let answer = array.sort((a, b) => a - b)[0];
    let mindiff = Math.abs(answer - n);
    
    for (let i=1; i<array.length; i++) {
        const diff = Math.abs(array[i] - n);
        if (diff < mindiff) {
            answer = array[i];
            mindiff = diff;
        }
    }
    
    return answer;
}
